### PR TITLE
Resolve route basename

### PR DESCRIPTION
**Pull request by igoroctaviano**
- Possible solution to #36

### CODE REVIEW

**Comment by hackermd** [comment]
@igoroctaviano could you please rebase the `fix/route-basename` branch on HEAD of the `master` branch?

File: public/config/local.js [hunk 1]
```diff
@@ -1,45 +1,46 @@
 window.config = {
   path: "/",
+  routerBasename: "/slim",
```
Comment (hackermd) [inline public/config/local.js:3]: Do we want this in this configuration file? Let's not stuff everything into `config/local.js`. This configuration file is only intended for local development and testing with the locally deployed containerized DCM4CHE archive.

File: src/AppConfig.d.ts [hunk 2]
```diff
@@ -41,5 +41,6 @@ export default interface AppConfig {
   path: string
   annotations: AnnotationSettings[]
   organization?: string
-  oidc?: OidcSettings
+  oidc?: OidcSettings,
+  routerBasename?: string
```
Comment (hackermd) [inline src/AppConfig.d.ts:45]: The existing parameter `path` was supposed to be used to configure the `BrowserRouter.basename`. Please remove the `routerBasename` parameter and use `path` instead.

**Review by hackermd — CHANGES_REQUESTED** [review]

**Comment by hackermd** [comment]
@igoroctaviano I could also include these changess in #52. I have moved the router components around and we may get a merge conflict anyways. What do you think?

**Comment by igoroctaviano** [comment]
> @igoroctaviano I could also include these changess in #52. I have moved the router components around and we may get a merge conflict anyways. What do you think?

Yep, please include these changes in #52 instead.

**Comment by hackermd** [comment]
Closing this PR in favor of #52.

**Pull request closed without merging**